### DISCUSSION TITLE
播放音乐时无法显示图片

### DISCUSSION THREAD

**Discussion by xlenco** [spencerwooo/onedrive-vercel-index#557] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/75837117/159150323-dcc332c2-120d-42d7-a5d4-b8c8b9a71568.png)
![image](https://user-images.githubusercontent.com/75837117/159150372-42de8622-f87c-40d7-a3d2-8dc8665628ff.png)

图片已经和音频放在同一目录了
我的站点：https://xilej.eu.org/%F0%9F%8E%B5%E9%9F%B3%E4%B9%90/

**Comment by spencerwooo** [comment, marked as answer]
专辑图片不是这样索引的，是根据音乐 metadata 索引的，你需要确保 OneDrive 正确识别到了你的专辑图片。

**Reply by xlenco** [reply]
已解决不过如果能显示歌词就更好了
![image](https://user-images.githubusercontent.com/75837117/159334190-fdf8b9e9-afe0-4e4f-ba37-b666fdb42acb.png)

**Reply by YAOmeihah** [reply]
请问一下你的  网页版onedrive 打开音乐的时候识别到专辑图片了嘛

**Reply by xlenco** [reply]
第一开始没有，后来我发现是因为我的专辑不带封面，然后给它嵌入个封面就可以了。不过你遇到的问题我没有遇到过。**可以识别到专辑封面**
你的可能是遇到bug了，等作者来解决吧！

**Reply by YAOmeihah** [reply]
好像不是程序的问题欸  是测试了一下其他账号的onedrive网页版是否能正常识别专辑图片
![image](https://user-images.githubusercontent.com/51868611/167837300-11cf173c-1c5b-419f-a3c5-973cdc72c43c.png)
上面这个是我自己的账号是可以识别的
下面这个是世纪互联的账号
![image](https://user-images.githubusercontent.com/51868611/167837376-f839e25b-9215-42aa-8d66-fc2ba1936ff4.png)
我的天啊  这是为啥哇

**Reply by YAOmeihah** [reply]
请问一下，你是 世纪互联的账号嘛  如果你是世纪互联的账号你可不可以截一下onedrive 网页版 能正常识别封面的截图嘛

**Comment by YAOmeihah** [comment]
请问是怎么解决的呢，我的onedrive 已经正常识别专辑图片 但是还是不能显示图片
![7e69cc728a5d5341dc761967ead20c9](https://user-images.githubusercontent.com/51868611/167787918-9a96ef14-8d57-412c-aa8a-e318aa43da62.jpg)
![image](https://user-images.githubusercontent.com/51868611/167787851-ea256dea-d0cc-4333-b2c1-edf99c0d697b.png)
链接
https://drive.hrlni.cn/zh-CN/Music/Timber%20-%20Pitbull%26Kesha/Timber%20-%20Pitbull%26Kesha.flac

**Reply by YAOmeihah** [reply]
![image](https://user-images.githubusercontent.com/51868611/167790378-8d0aad9a-58db-467b-b067-e4aa7c11c817.png)

**Reply by YAOmeihah** [reply]
显示该项目没有有效的缩略图，我试着下载别人可以正常识别缩略图的音乐，上传后但是还是显示无法正常显示

**Reply by xlenco** [reply]
我是直接将图片嵌入弄好的https://xilej.eu.org/%F0%9F%8E%B5%E9%9F%B3%E4%B9%90/aLIEz%20-%20mizuki%20(%E7%91%9E%E8%91%B5)%26SawanoHiroyuki%5BnZk%5D.flac

**Reply by YAOmeihah** [reply]
我下载你这个音乐上传试一下

**Reply by YAOmeihah** [reply]
![image](https://user-images.githubusercontent.com/51868611/167806572-bcb14d3b-6d79-43d0-a87b-eaad26155bdc.png)
这是为啥呢

**Reply by YAOmeihah** [reply]
还是显示无法显示有效的缩略图

**Reply by YAOmeihah** [reply]
https://drive.hrlni.cn/zh-CN/Music/aLIEz%20-%20mizuki%20(%E7%91%9E%E8%91%B5)%26SawanoHiroyuki%5BnZk%5D/aLIEz%20-%20mizuki%20(%E7%91%9E%E8%91%B5)%26SawanoHiroyuki%5BnZk%5D.flac

**Reply by xlenco** [reply]
可能遇到bug了，建议等作者回复。

**Comment by YAOmeihah** [comment]
@spencerwooo
